vgen/src/main/java/io/gapi/vgen/php/PhpContext.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.php;
+
+/**
+ * A PhpContext provides functionality specific to a context in PHP.
+ */
+public interface PhpContext {
+  void resetState(PhpSnippetSet<?> phpSnippetSet, PhpContextCommon phpCommon);
```

Comment:
The equivalent class in Java codegen (JavaContext) doesn't have this commented either. I'd prefer to make a documentation pass in a separate PR.